test/fixtures/responses/newOrderBook.json

```diff
@@ -0,0 +1 @@
Account":"rDbsCJr5m8gHDCNEHCZtFxcXHsD4S9jH83","BookDirectory":"6EAB7C172DEFA430DBFAD120FDC373B5F5AF8B191649EC98570C560B764D760C","BookNode":"0000000000000000","Flags":131072,"LedgerEntryType":"Offer","OwnerNode":"0000000000000001","PreviousTxnID":"9A0B6B76F0D86614F965A2FFCC8859D8607F4E424351D4CFE2FBE24510F93F25","PreviousTxnLgrSeq":10708382,"Sequence":110061,"TakerGets":{"currency":"BTC","issuer":"rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B","value":"0.003768001830745216"},"TakerPays":{"currency":"USD","issuer":"rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B","value":"1.308365894430151"},"index":"B971769686CE1B9139502770158A4E7C011CFF8E865E5AAE5428E23AAA0E146D","owner_funds":"0.2229210189326514","quality":"347.2306949944844"},{"Account":"rDVBvAQScXrGRGnzrxRrcJPeNLeLeUTAqE","BookDirectory":"6EAB7C172DEFA430DBFAD120FDC373B5F5AF8B191649EC98570C87DF25DC4FC6","BookNode":"0000000000000000","Expiration":472783298,"Flags":131072,"LedgerEntryType":"Offer","OwnerNode":"00000000000003D2","PreviousTxnID":"E5F9A10F29A4BB3634D5A84FC96931E17267B58E0D2D5ADE24FFB751E52ADB9E","PreviousTxnLgrSeq":10713533,"Sequence":35788,"TakerGets":{"currency":"BTC","issuer":"rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B","value":"0.5"},"TakerPays":{"currency":"USD","issuer":"rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B","value":"176.3546101589987"},"index":"D2CB71038AD0ECAF4B5FF0A953AD1257225D0071E6F3AF9ADE67F05590B45C6E","owner_funds":"6.617688680663627","quality":"352.7092203179974"},{"Account":"rN6jbxx4H6NxcnmkzBxQnbCWLECNKrgSSf","BookDirectory":"6EAB7C172DEFA430DBFAD120FDC373B5F5AF8B191649EC98570CC0B8E0E2C000","BookNode":"0000000000000000","Flags":131072,"LedgerEntryType":"Offer","OwnerNode":"0000000000000000","PreviousTxnID":"2E16ACFEAC2306E3B3483D445787F3496FACF9504F7A5E909620C1A73E2EDE54","PreviousTxnLgrSeq":10558020,"Sequence":491,"TakerGets":{"currency":"BTC","issuer":"rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B","value":"0.5"},"TakerPays":{"currency":"USD","issuer":"rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B","value":"179.48"},"index":"DA853913C8013C9471957349EDAEE4DF4846833B8CCB92008E2A8994E37BEF0D","owner_funds":"0.5","quality":"358.96","taker_gets_funded":{"currency":"BTC","issuer":"rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B","value":"0.499001996007984"},"taker_pays_funded":{"currency":"USD","issuer":"rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B","value":"179.1217564870259"}},{"Account":"rDVBvAQScXrGRGnzrxRrcJPeNLeLeUTAqE","BookDirectory":"6EAB7C172DEFA430DBFAD120FDC373B5F5AF8B191649EC98570CD2F24C9C145D","BookNode":"0000000000000000","Expiration":472783299,"Flags":131072,"LedgerEntryType":"Offer","OwnerNode":"00000000000003D2","PreviousTxnID":"B1B12E47043B4260223A2C4240D19E93526B55B1DB38DEED335DACE7C04FEB23","PreviousTxnLgrSeq":10713534,"Sequence":35789,"TakerGets":{"currency":"BTC","issuer":"rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B","value":"0.8"},"TakerPays":{"currency":"USD","issuer":"rvYAfWj5gh67oV6fW32ZzP3Aw4Eubs59B","value":"288.7710263794967"},"index":"B89AD580E908F7337CCBB47A0BAAC6417EF13AC3465E34E8B7DD3BED016EA833","quality":"360.9637829743709"}]}
```

Comment:
Set this up like the other JSONs in this folder (with all the spacing and lines), so it's easier to read